welcome/cloud-experts-rosa-hcp-sts-explained.adoc

```diff
@@ -0,0 +1,116 @@
+* The service cannot do anything outside of those permissions.
+* There is no need to rotate or revoke credentials. Whenever the service needs to perform an action, it obtains credentials that expire in one hour or less. Credential expiration reduces the risks of credentials leaking and being reused.
+
+{hcp-title} grants users least-privilege permissions with short-term security credentials to specific and segregated IAM roles. The credentials are associated with IAM roles specific to each component and cluster that makes AWS API calls. This method aligns with principles of least-privilege and secure practices in cloud service resource management.
+
+[id="components-specific-to-rosa-hcp-with-sts"]
+== Components of {hcp-title}
+* *AWS infrastructure* - The infrastructure required for the cluster including the EC2 instances, storage, and networking components. See xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-aws-compute-types_rosa-service-definition[AWS compute types] to see the supported instance types for compute nodes and xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-ec2-instances_rosa-sts-aws-prereqs[provisioned AWS infrastructure] for more information on control plane and infrastructure node configuration.
+* *AWS STS* - A method for granting short-term, dynamic tokens to provide users the necessary permissions to temporarily interact with your AWS account resources.
+* *OpenID Connect (OIDC)* - A mechanism for cluster Operators to authenticate with AWS, assume the cluster roles through a trust policy, and obtain temporary credentials from STS to make the required API calls.
+* *Roles and policies* - The roles and policies used by {hcp-title} can be divided into AWS-managed account-wide roles and policies and Operator roles and policies.
++
+The policies determine the allowed actions for each of the roles. See xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-about-iam-resources[About IAM resources for ROSA clusters that use STS] for more details about the individual roles and policies.
++
+** The AWS-managed account-wide roles are:
++
+*** ManagedOpenShift-Installer-Role
+*** ManagedOpenShift-Worker-Role
+*** ManagedOpenShift-Support-Role
++
+** The AWS-managed account-wide policies are:
++
+*** ManagedOpenShift-Installer-Role-Policy
+*** ManagedOpenShift-Worker-Role-Policy
+*** ManagedOpenShift-Support-Role-Policy
+*** ManagedOpenShift-openshift-ingress-operator-cloud-credentials ^[1]^
+*** ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent ^[1]^
+*** ManagedOpenShift-openshift-cloud-network-config-controller-cloud ^[1]^
+*** ManagedOpenShift-openshift-machine-api-aws-cloud-credentials ^[1]^
+*** ManagedOpenShift-openshift-cloud-credential-operator-cloud-crede ^[1]^
+*** ManagedOpenShift-openshift-image-registry-installer-cloud-creden ^[1]^
```

Comment:
AWS managed policies are not naming as above. As an example the managed policy for installer role is named "ROSAInstallerPolicy".

Reply:
Please let me know if the policies are correct. Thank you!

Reply:
That looks good